src/components/Comments/ReplyCommentItem.js

```diff
@@ -46,6 +55,15 @@ const ReplyCommentItem = ({
     comment.data.count_upvote - comment.data.count_downvote
   );
   const [statusVote, setStatusVote] = React.useState('');
+  const [loading, setLoading] = React.useState({
+    loadingDm: false,
+    loadingDmAnon: false
+  });
+
+  const [profile] = React.useContext(Context).profile;
+  const {createSignChat} = useCreateChat();
+  const {sendMessageDM} = useDMMessage();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New state variables and hooks have been introduced for handling direct messages. The use of local state for optimistic UI is a good practice. However, error handling should be considered in case the operations fail.



<!-- This is an auto-generated comment by OSS CodeRabbit -->